src/gui/windows/debug.zig

```diff
@@ -25,7 +25,7 @@ pub var window = GuiWindow {
 pub fn render() void {
 	draw.setColor(0xffffffff);
 	var y: f32 = 0;
-	draw.print("    fps: {d:.0} Hz{s}", .{1.0/main.lastFrameTime.load(.monotonic), if(main.settings.vsync) @as([]const u8, " (vsync)") else ""}, 0, y, 8, .left);
+	draw.print("    fps: {d:.0} Hz{s}", .{1.0/main.lastDeltaTime.load(.monotonic), if(main.settings.vsync) @as([]const u8, " (vsync)") else ""}, 0, y, 8, .left);
```

Comment:
Could you maybe also add the fps limit in parenthesis, if the fps is limited?

Reply:
Changed it to either
```
a) FPS Limit: N
b) FPS: Unlimited
```